src/components/manager.js

```diff
@@ -179,6 +180,26 @@ export class Manager extends Component {
     window.addEventListener('storage', this._goToSlide);
     window.addEventListener('keydown', this._handleKeyPress);
     window.addEventListener('resize', this._handleScreenChange);
+    if (
+      navigator &&
+      navigator.presentation &&
+      navigator.presentation.receiver
+    ) {
+      navigator.presentation.receiver.connectionList.then(list => {
```

Comment:
You need to test for `connectionList` and `then` too...

Maybe:

```js
if (((((navigator || {}).presentation || {}).receiver || {}).connectionList) {
```

(assume set connectionList implies thennable)